src/engraving/layout/v0/systemlayout.h

```diff
@@ -51,14 +51,16 @@ class SystemLayout
     static void restoreLayout2(System* system, LayoutContext& ctx);
     static void setMeasureHeight(System* system, double height, LayoutContext& ctx);
     static void layoutBracketsVertical(System* system, LayoutContext& ctx);
-    static void layoutInstrumentNames(System* system);
+    static void layoutInstrumentNames(System* system, LayoutContext& ctx);
 
     static void setInstrumentNames(System* system, LayoutContext& ctx, bool longName, Fraction tick = { 0, 1 });
 
+    static double minDistance(LayoutContext& ctx, const System* top, const System* bottom);
```

Comment:
LayoutContext& ctx should probably be the last argument here (for consistency)

Reply:
In a next PR I will fix it